osu.Game/OsuGame.cs

```diff
@@ -214,7 +214,7 @@ private bool globalHotkeyPressed(InputState state, KeyDownEventArgs args)
                 }
             }
 
-            return base.OnKeyDown(state, args);
+            return OnKeyDown(state, args);

```

Comment:
Leave for now. Just something for the next person who can look into it to consider. Just commented so it doesn't get lost.